devtools/gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
-#Tue Nov 05 11:34:08 BRST 2019
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+#Tue Nov 26 00:14:48 BRST 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```

Comment:
I wonder if we should just be using the `-bin` variant instead of `-all`. Not that it matters much :)

Reply:
Intellij changed it so that Gradle sources are available

Reply:
But do we need them? I guess they help when developing the plugin...

Reply:
Yes it helped me a lot to debug and understand how some Gradle features work.